src/components/Error.tsx

```diff
@@ -0,0 +1,38 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../context/TodoContext';
+
+// enum ErrorMessages {
+//   'Unable to load todos',
+//   'Title should not be empty',
+//   'Unable to add a todo',
+//   'Unable to delete a todo',
+//   'Unable to update a todo',
+// }
+
+export const Error = () => {
+  const { handleCloseError, error } = useContext(TodosContext);
+
+  const handleButtonClick = () => {
+    handleCloseError();
+  };
+
```

Comment:
add useEffect with setTimeout here 
```suggestion
 useEffect(() => {
    setTimeout(() => {
      setError(ErrorType.None);
    }, 3000);
  }, [error, setError]);
```